lua/lualine/extensions/quickfix.lua

```diff
@@ -18,13 +18,24 @@ end
 
 local M = {}
 
-function M.init()
+function M.get_colours()
   -- Make sure ft wf doesn't create a custom statusline
   vim.g.qf_disable_statusline = true
+  return {
```

Comment:
the qf_disable_statusline still only needs to be called on init. I asked you to move the part you added to different location.

Reply:
Oops sorry, my bad, I did not understand. I have now moved the colours assignment to its own location, decoupling it from `M.init()`.